src/System.Management.Automation/CoreCLR/CorePsAssemblyLoadContext.cs

```diff
@@ -1,19 +1,14 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@iSazonov, your last commit had 2 failures in `PowerShell-CI-windows`
Can load a native assembly.Can load native dll
```powershell
Expected $null or empty, but got C:\Users\VssAdministrator\AppData\Local\Temp\tmpCA73.tmp\testDllNativeFolder\win-x64;C:\Users\VssAdministrator\AppData\Local\Temp\tmpCA73.tmp\testDllNativeFolder\managed.dll.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Basic\Assembly.LoadNative.Tests.ps1: line 64
64:         (dir $root)  -join ";" | Should -BeNullOrEmpty
```
Can load a native assembly.Can load native dll
```powershell
Unable to find type [TestNativeClass].
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Basic\Assembly.LoadNative.Tests.ps1: line 76
76:         [TestNativeClass]::LoadNative() | Should -Not -BeNullOrEmpty

```